lib/modules/manager/nuget/extract/global-manifest.ts

```diff
@@ -54,5 +56,5 @@ export function extractMsbuildGlobalManifest(
     }
   }
 
-  return { deps };
+  return { deps, extractedConstraints };
```

Comment:
```suggestion
  return { deps, extractedConstraints && (...{ extractedConstraints }) };
```
so we don't have a property with an undefined value. not sure about the braces.